### PR TITLE
document: update formats

### PR DESCRIPTION
**Pull request by vankichi**
### Description:

I have updated the format of some documents for more readability.

### Environment:

- Go Version: 1.18.1
- Docker Version: 20.10.8
- Kubernetes Version: 1.22.0
- NGT Version: 1.14.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [ ] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [ ] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by hlts2 — APPROVED** [review]
LGTM!

File: docs/api/build_proto.md [hunk 1]
```diff
@@ -72,14 +72,14 @@ There are many tools for building proto in Rust, we use [tonic](https://github.c
 1.  Create project
 
     ```bash
-    $ cargo new --lib vald-grpc
```
Comment (kevindiu) [inline docs/api/build_proto.md:75]: I think it is better to leave `$` here for easy understanding on if it is command or not.

**Review by kpango — APPROVED** [review]
LGTM

File: docs/api/build_proto.md (hunk 1 above)
Comment (vankichi) [reply docs/api/build_proto.md:75]: It is for user can run command only Copy and Paste

File: docs/api/build_proto.md [hunk 2]
```diff
@@ -72,14 +72,14 @@ There are many tools for building proto in Rust, we use [tonic](https://github.c
...
     ```
 
 1.  Edit `Cargo.toml`
 
     ```bash
-    $ cd vald-grpc
-    $ vim Cargo.toml
+    cd vald-grpc && \
+    vim Cargo.toml
     ---
```
Comment (kevindiu) [inline docs/api/build_proto.md:83]: I think we should separate this in another block

File: docs/api/build_proto.md [hunk 3]
```diff
@@ -108,31 +108,37 @@ There are many tools for building proto in Rust, we use [tonic](https://github.c
 
     ```bash
     // create proto file root dir
```
Comment (kevindiu) [inline docs/api/build_proto.md:110]: same
https://github.com/vdaas/vald/pull/1634/files#r858184192

File: docs/api/build_proto.md (hunk 2 above)
Comment (vankichi) [reply docs/api/build_proto.md:83]: All this kind of changes is for the clipboard function in the public document.

**Pull request merged**